tests/backup/backup_dr_test.go

```diff
@@ -0,0 +1,340 @@
+				backupName = fmt.Sprintf("%s-%v", BackupNamePrefix, time.Now().Unix())
+				appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, []string{namespace})
+				err = CreateBackupWithValidation(ctx, backupName, SourceClusterName, bkpLocationName, backupLocationUID, appContextsToBackup, labelSelectors, BackupOrgID, srcClusterUid, preRuleName, preRuleUid, postRuleName, postRuleUid)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of backup [%s]", backupName))
+				backupNames = append(backupNames, backupName)
+				backupNamespaceMap[backupName] = namespace
+			}
+		})
+
+		Step("Taking manual backup of namespaces with rules from the DR site", func() {
+			log.InfoD(fmt.Sprintf("Taking manual backup of namespaces with rules from the DR site"))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, namespace := range bkpNamespaces {
+				backupName = fmt.Sprintf("%s-%v", BackupNamePrefix, time.Now().Unix())
+				appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, []string{namespace})
+				err = CreateBackupWithValidation(ctx, backupName, DestinationClusterName, bkpLocationName, backupLocationUID, appContextsToBackup, labelSelectors, BackupOrgID, destClusterUid, preRuleName, preRuleUid, postRuleName, postRuleUid)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of backup [%s]", backupName))
+				backupNames = append(backupNames, backupName)
+				drBackupNames = append(drBackupNames, backupName)
+				backupNamespaceMap[backupName] = namespace
+			}
+		})
+
+		Step("Taking schedule backup of namespaces with rules from the DC site", func() {
+			log.InfoD(fmt.Sprintf("Taking schedule backup of namespaces with rules from the DC site"))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, namespace := range bkpNamespaces {
+				scheduleName := fmt.Sprintf("%s-dc-schedule-with-rules-%s", BackupNamePrefix, RandomString(4))
+				log.InfoD("Creating a schedule backup of namespace [%s] without pre and post exec rules", namespace)
+				appContextsToBackup := FilterAppContextsByNamespace(scheduledAppContexts, []string{namespace})
+				scheduleBackupName, err := CreateScheduleBackupWithValidation(ctx, scheduleName, SourceClusterName, bkpLocationName, backupLocationUID, appContextsToBackup,
+					labelSelectors, BackupOrgID, "", "", "", "", periodicSchedulePolicyName, periodicSchedulePolicyUid)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of backup [%s]", scheduleBackupName))
+				err = SuspendBackupSchedule(scheduleName, periodicSchedulePolicyName, BackupOrgID, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Suspending Backup Schedule [%s] ", scheduleName))
+				backupNames = append(backupNames, scheduleBackupName)
+				scheduleNames = append(scheduleNames, scheduleName)
+				backupNamespaceMap[scheduleBackupName] = namespace
+			}
```

Comment:
Since we have taken backup from DC cluster, can we add one more step to restore it back to DC again

Reply:
Done, Added a step for DC restore.